src/v1_services/compare_service.py

```
@@ -0,0 +1,329 @@
+    country_map = {c.country_id: c.country_name for c in countries}
+
+    # ----- Step 2: 질문을 벡터로 변환 (임베딩) -----
+    # 질문 임베딩은 1회만 수행하고, 두 국가 검색에 모두 같은 벡터를 사용합니다.
+    # ※ v1에서는 한국어 질문을 그대로 임베딩 (v2에서는 영어 번역 후 임베딩)
+    try:
+        text_input = TextEmbeddingInput(text=query, task_type="RETRIEVAL_QUERY")
+        embeddings = embedding_model.get_embeddings([text_input])
+        query_vector = embeddings[0].values
+    except Exception as e:
+        print(f"❌ 임베딩 실패: {e}")
+        raise e
```

Ruff (0.15.2)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)